examples/machine-learning/a3-megagpu-8g/slurm-a3mega-image.yaml

```diff
@@ -0,0 +1,305 @@
+      - 10.0.0.*
+      - 10.1.0.*
+      - 10.2.0.*
+      - 10.3.0.*
+      - 10.4.0.*
+      - 10.5.0.*
+      - 10.6.0.*
+      - 10.7.0.*
+      - $(vars.slurm_cluster_name)*
+      docker:
+        enabled: true
+        world_writable: true
+      runners:
+      - type: data
+        destination: /etc/apt/preferences.d/block-broken-nvidia-container
+        content: |
+          Package: nvidia-container-toolkit nvidia-container-toolkit-base libnvidia-container-tools libnvidia-container1
+          Pin: version 1.17.7-1
+          Pin-Priority: 100
+      # it is important that kernel upgrades do not occur before running the
+      # solution for building Slurm (which doesn't handle them well on the fly)
+      # if you follow this rule, any module which supports DKMS will be
+      # properly configured at the end of image building (gVNIC, NVIDIA, ...)
+      - type: shell
+        destination: prevent_unintentional_upgrades.sh
+        content: |
+          #!/bin/bash
+          # Unattended upgrades are disabled in this blueprint so that software does not
+          # get updated daily and lead to potential instability in the cluster environment.
+          #
+          # Unattended Upgrades installs available security updates from the Ubuntu
+          # security pocket for installed packages daily by default. Administrators who
+          # disable this feature assume all responsibility for manually reviewing and
+          # patching their systems against vulnerabilities.
+          #
+          # To enable unattended upgrades, please remove the following lines:
+          #   systemctl stop unattended-upgrades.service
+          #   systemctl disable unattended-upgrades.service
+          #   systemctl mask unattended-upgrades.service
+          set -e -o pipefail
+          systemctl stop unattended-upgrades.service
+          systemctl disable unattended-upgrades.service
+          systemctl mask unattended-upgrades.service
+          apt-mark hold google-compute-engine
+          apt-mark hold google-compute-engine-oslogin
+          apt-mark hold google-guest-agent
+          apt-mark hold google-osconfig-agent
+      - type: data
+        destination: /var/tmp/slurm_vars.json
+        content: |
+          {
+            "reboot": false,
+            "install_cuda": false,
+            "install_ompi": true,
+            "install_lustre": false,
+            "install_managed_lustre": false,
+            "install_gcsfuse": true,
+            "monitoring_agent": "cloud-ops",
+            "use_open_drivers": true
+          }
+      - type: shell
+        destination: install_slurm.sh
+        content: |
+          #!/bin/bash
+          set -e -o pipefail
+          apt-get update
+          apt-get install -y git
+          ansible-galaxy role install googlecloudplatform.google_cloud_ops_agents
+          ansible-pull \
+              -U https://github.com/GoogleCloudPlatform/slurm-gcp -C 6.9.1 \
```

Comment:
Please create a follow-up task to update to the latest Slurm-GCP tag (believe it's 6.10.0). Right now we are doing an in-place re-factor so I'm happy to leave it as is.